hazelcast-jet-all/src/test/java/com/hazelcast/jet/server/JetCommandLineTest.java

```diff
@@ -483,6 +484,72 @@ public void test_submit_argsPassing() {
         assertTrueEventually(() -> assertContains(captureOut(), " with arguments [--jobOption, fooValue]"));
     }
 
+    @Test
+    public void testTargetsMixin() {
+        String target = "foobar@127.0.0.1:5701,127.0.0.1:5702";
+
+        testTargetsCommand("cancel", "-t", target, "jobName");
+        testTargetsCommand("-t", target, "cancel", "jobName");
+
+        testTargetsCommand("cluster", "-t", target);
+        testTargetsCommand("-t", target, "cluster");
+
+        testTargetsCommand("delete-snapshot", "snapshotName", "-t", target);
+        testTargetsCommand("-t", target, "delete-snapshot", "snapshotName");
+
+        testTargetsCommand("list-jobs", "-t", target);
+        testTargetsCommand("-t", target, "list-jobs");
+
+        testTargetsCommand("list-snapshots", "-t", target);
+        testTargetsCommand("-t", target, "list-snapshots");
+
+        testTargetsCommand("restart", "jobName", "-t", target);
+        testTargetsCommand("-t", target, "restart", "jobName");
+
+        testTargetsCommand("resume", "jobName", "-t", target);
+        testTargetsCommand("-t", target, "resume", "jobName");
+
+        testTargetsCommand("save-snapshot", "jobName", "snapshotName", "-t", target);
+        testTargetsCommand("-t", target, "save-snapshot", "jobName", "snapshotName");
+
+        testTargetsCommand("submit", "-t", target, testJobJarFile.toString());
+        testTargetsCommand("-t", target, "submit", testJobJarFile.toString());
+
+        testTargetsCommand("suspend", "jobName", "-t", target);
+        testTargetsCommand("-t", target, "suspend", "jobName");
+    }
+
+    @Test
+    public void testTargetsPrecedence() {
```

Comment:
Use more descriptive name. It is not immediately obvious if the option before or after command takes precedence.